salt/environment_settings.yml

```diff
@@ -141,19 +149,22 @@ environments:
           edx-worker:
             number: 1
             type: t2.medium
-      sandbox:
+      sandbox-residential-live:
         business_unit: operations
         domains:
-          cms: studio-micromasters.d.mitx.mit.edu
-          lms: micromasters.d.mitx.mit.edu
-          preview: preview-micromasters.d.mitx.mit.edu
+          cms: studio-mitx-qa-sandbox.mitx.mit.edu
+          lms: mitx-qa-sandbox.mitx.mit.edu
+          preview: preview-mitx-qa-sandbox.mitx.mit.edu
+          gitreload: gr-qa-sandbox.mitx.mit.edu
         versions:
-          edxapp: master
-          forum: master
+          <<: *sandbox_residential_versions
         instances:
           edx:
             number: 1
             type: t2.medium
+          edx-worker:
```

Comment:
There's no need to specify the edx worker instance count because the worker will be colocated with the edxapp on the same box.